SubEthaEdit-Mac/Source/TCMFoundation/TCMBencodingUtilities.m

```diff
@@ -173,17 +169,14 @@ id TCM_CopyBdecodedObject(uint8_t *aBytes, unsigned *aPosition, unsigned aLength
 					if (value) {
 						NSString *decodedKey = [S_bencodingDictionaryKeysDictionary objectForKey:key];
 						if (! decodedKey) {
-							decodedKey = [[key retain] autorelease];
+							decodedKey = key;
 							[S_bencodingDictionaryKeysDictionary setObject:decodedKey forKey:key];
 						}
 						[result setObject:value forKey:decodedKey];
-						[value release];
 						value = nil;
 					}
-					[key release];
 					key = nil;
```

Comment:
the = nil can go too as the variables are scoped correctly here. same for value.